src/utils/fluid_conv.c

```diff
@@ -21,6 +21,12 @@
 #include "fluid_conv.h"
 
 
+/* EMU 8k/10k don't follow spec in regards to volume attenuation.
+ * This factor is used in the equation pow (10.0, cb / FLUID_ATTEN_POWER_FACTOR).
+ * By the standard this should be -200.0. */
+/* 07/11/2008 modified by S. Christian Collins for increased velocity sensitivity.  Now it equals the response of EMU10K1 programming.*/
```

Comment:
break long line?

Reply:
clang-format (#206) should take care of that some when in the future.